kubernetes/oci-cloud-controller/datasources.tf

```diff
@@ -0,0 +1,24 @@
+
+data "template_file" "oci-cloud-controller-secret" {
+  template = "${file("${path.module}/cloud-provider-secret.yaml")}"
+  vars = {
+    tenancy     = "${var.tenancy}"
+    region      = "${var.region}"
+    compartment = "${var.compartment_ocid}"
+    
+    # if cloud_controller_user_ocid is empty string then generate a user/group/policy and use it here
+    fingerprint = "${var.cloud_controller_user_ocid=="" ? join("", oci_identity_api_key.cloud_controller_key_assoc.*.fingerprint) : var.cloud_controller_user_fingerprint }"
+    user        = "${var.cloud_controller_user_ocid=="" ? join("", oci_identity_user.cloud_controller_user.*.id) : var.cloud_controller_user_ocid }"
+    key         = "${var.cloud_controller_user_ocid=="" ? jsonencode(join("", tls_private_key.cloud_controller_user_key.*.private_key_pem)) : jsonencode(file(var.cloud_controller_user_private_key_path))}"
+    
+    subnet1 = "${var.subnet1}"
+    subnet2 = "${var.subnet1}"
```

Comment:
Typo - should read `subnet2 = "${var.subnet2}"`

Reply:
This is actually wrong and I've misunderstood the CCM LB requirements. I need to create some subnets for the load balancers to live on.